### PR TITLE
Show average and floor price, if available

### LINKED ISSUE #2683: Should show more NFT properties
On iOS it shows `floor price`, `average price` and other properties, but Android doesn't show.

IMG_5259
IMG_5258

### PR DESCRIPTION
**Pull request by justindg**
- Closes #2683

### CODE REVIEW

**Review by JamesSmartCell — CHANGES_REQUESTED** [review]

**Comment by JamesSmartCell** [comment]
@justindg what are we using the collection call for? Can we get this info from the main opensea call? Or is this for the times when for some reason we don't have the opensea data?

**Comment by justindg** [comment]
> @justindg what are we using the collection call for? Can we get this info from the main opensea call? Or is this for the times when for some reason we don't have the opensea data?

The main opensea call (`/asset/{address}`) contains the `average_price`, but for some reason it returns a `0` value for `floor_price`, hence I chained the collection call where we could find more accurate data.

File: app/src/main/java/com/alphawallet/app/viewmodel/TokenFunctionViewModel.java [hunk 1]
```diff
@@ -820,9 +862,18 @@ private void onAsset(String result, Token token, BigInteger tokenId)
                 if (!TextUtils.isEmpty(asset.getImage()))
                 {
                     loadedFromApi = true;
-                    storeAsset(token, tokenId, new NFTAsset(result), oldAsset);
-                    asset.attachOpenSeaAssetData(osAsset);
-                    nftAsset.postValue(asset);
+
```
[the remark this answers is not in the document]
Comment (justindg) [reply app/src/main/java/com/alphawallet/app/viewmodel/TokenFunctionViewModel.java:865]: I store the asset even if the collection check fails, I moved it to the end of `onCollection()` method

[the remark this answers is not in the document]
Comment (JamesSmartCell) [reply app/src/main/java/com/alphawallet/app/viewmodel/TokenFunctionViewModel.java:865]: > I store the asset even if the collection check fails, I moved it to the end of `onCollection()` method

Yes, that would be better I think. Also ensure that the screen is refreshed if required, even if collection fails. Might be an edge condition but worth doing.

**Review by JamesSmartCell — APPROVED** [review]

**Pull request merged**